src/models/note.ts

```diff
@@ -12,6 +17,8 @@ export interface INote extends Document {
 const schema = new Schema({
   sender: { type: mongoose.Schema.Types.ObjectId, ref: 'User', required: true},
   receiver: { type: mongoose.Schema.Types.ObjectId, ref: 'User', required: true },
+  senderSlackID: { type: String, required: true },
+  receiverSlackID: { type: String, required: true },
```

Comment:
These Ds should be lower case.

Reply:
Actually, not sure this data should be in the notes object. It's information that relates to a user, not the note itself.